ciao-cli/tenant.go

```diff
@@ -210,3 +388,61 @@ func listTenantResources(t *template.Template) error {
 
 	return nil
 }
+
+func listTenantConfig(t *template.Template, tenantID string) error {
+	if tenantID == "" {
+		fatalf("Missing required -for-tenant parameter")
+	}
+
+	config, err := getCiaoTenantConfig(tenantID)
+	if err != nil {
+		fatalf(err.Error())
+	}
+
+	if t != nil {
+		if err := t.Execute(os.Stdout, &config); err != nil {
+			fatalf(err.Error())
+		}
+		return nil
+	}
+
+	fmt.Printf("Tenant [%s]\n", tenantID)
+	fmt.Printf("\tName: %s\n", config.Name)
+	fmt.Printf("\tSubnetBits: %d\n", config.SubnetBits)
+
+	return nil
+}
+
+func listAllTenants(t *template.Template) error {
+	var tenants types.TenantsListResponse
+
+	url, err := getCiaoTenantsResource()
+	if err != nil {
+		fatalf(err.Error())
+	}
+
+	resp, err := sendCiaoRequest("GET", url, nil, nil, api.TenantsV1)
+	if err != nil {
+		fatalf(err.Error())
+	}
+
+	err = unmarshalHTTPResponse(resp, &tenants)
+	if err != nil {
+		return err
```

Comment:
Any reason not to use fatalf here?  I guess it doesn't really make much difference and this code is all going to have to change but it's a little inconsistent.

Reply:
I'll change that.